src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -327,8 +327,14 @@ private static bool ShouldRemoveHttpClient(SocketException exception)
 
         private static readonly Exception ServerCertificateCustomValidationCallbackRegistrationException;
 
+        private static readonly bool? CheckCertificateRevocationList;
+
         static RequestExecutor()
         {
+            var checkCertificateRevocationListAsString = Environment.GetEnvironmentVariable("RAVEN_CHECK_CERTIFICATE_REVOCATION_LIST");
```

Comment:
Why not make it part of the Raven Configuration - `SecurityConfiguration`? Then you'd get parsing env vars, cli params and settings all for free.

Reply:
Umm nvm, that's client-side